src/common/cid-log.test.ts

```diff
@@ -0,0 +1,130 @@
+import * as fc from 'fast-check'
+import CID from 'cids'
+import multihash from 'multihashing-async'
+import * as cidLog from './cid-log'
+import * as storage from '../storage'
+
+async function generateCids(data: Uint8Array[]): Promise<string[]> {
+  const promisedCids = data.map(async bytes => {
+    const mhash = await multihash(bytes, 'sha2-256')
```

Comment:
Oh my god. I needed this in one of my tests haha.

Reply:
Yeah! That's been super helpful for getting some real CIDs :smile: